marklogic-data-hub/src/main/java/com/marklogic/hub/impl/HubProjectImpl.java

```diff
@@ -646,6 +646,11 @@ public void exportProject(File location) {
         }
     }
 
+    @Override
```

Comment:
I commented on the JIRA ticket that I think we should get rid of this from the mockup. We don't have a concept of a "Project Name", and we shouldn't invent one during development. I think the requirement needs to be removed from the story. 

I can see an argument for the project directory, though I don't think that really needs to be there either.

Reply:
I saw your comment and response from @sbayatpur on it. The suggestion from her is to use proj dir name for now. I think we can change it once she decides on 'Project Name'.